.github/workflows/build.yml

```diff
@@ -0,0 +1,82 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Build
+
+on:
+  push:
+    branches:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: python:3.8
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Cache virtualenvironment
+        uses: actions/cache@v2
+        with:
+          path: ~/.venv
+          key: ${{ hashFiles('requirements.txt', 'requirements-dev.txt') }}
+
+      - name: Upgrade pip
+        run: pip install --upgrade pip
+
+      - name: Create and activate Virtualenv
+        run: |
+          pip install virtualenv
+          [ ! -d ".venv" ] && virtualenv .venv
+          . .venv/bin/activate
+
+      - name: Install dependencies
+        run: |
+          pip install -r requirements-dev.txt
```

Comment:
I think you'll need to install requirements.txt here too? You can do that in a single command 

`pip install -r requirements.txt requirements-dev.txt`